includes/templates/bootstrap/css/dist-client_styles.php

```diff
@@ -0,0 +1,7 @@
+/* 
```

Comment:
Churning ... since this is a PHP file (even though there's no initial content), please add a `<?php` as the first line and a `?>` after the comments.  I've found various PCI scanners flagging comments as 'bad', so also embellish the (now hidden) comments to indicate that any site-specific content be placed between the `<style>` and `</style>` tags.